types/node/ts3.6/base.d.ts

```diff
@@ -12,8 +12,6 @@
 /// <reference lib="esnext.intl" />
 /// <reference lib="esnext.bigint" />
 
-// Base definitions for all NodeJS modules that are not specific to any version of TypeScript:
```

Comment:
Keep?  (Might be copy+paste, but better to be conservative)

Reply:
Done :heavy_check_mark: